lib/gettext_i18n_rails/backend.rb

```diff
@@ -23,7 +23,12 @@ def translate(locale, key, options)
           interpolate(translation, options)
         else
           result = backend.translate(locale, key, options)
-          (RUBY19 and result.is_a?(String)) ? result.dup.force_encoding("UTF-8") : result
+          if RUBY19 && result.is_a?(String)
```

Comment:
can delete all the `if RUBY19`

Reply:
> RUBY19 cleanup can be it's own PR

On it's way.

Reply:
https://github.com/grosser/gettext_i18n_rails/pull/197